jetson/nav/README.md

```diff
@@ -1,156 +1,203 @@
-## LCM Channels Publishing/Subscribed To 
+## LCM Channels Publishing/Subscribed To
+
 **Target [subscriber]** \
-Messages: [ AutonState.lcm ](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/AutonState.lcm) “/auton_state” \
+Messages: [Enable.lcm](https://github.com/umrover/mrover-workspace/blob/main/rover_msgs/Enable.lcm)
+“/auton_enabled” \
 Publishers: jetson/teleop \
 Subscribers: jetson/nav
 
 **Course [subscriber]** \
-Messages: [ Course.lcm ](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/Course.lcm) “/course” \
+Messages: [Course.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/Course.lcm) “/course” \
 Publishers: simulators/nav, base_station/gui \
 Subscribers: jetson/nav
 
 **Obstacle [subscriber]** \
-Messages: [ Obstacle.lcm ](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/Obstacle.lcm) “/obstacle” \
+Messages: [Obstacle.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/Obstacle.lcm)
+“/obstacle” \
 Publishers: jetson/percep \
 Subscribers: jetson/nav
 
 **Odometry [subscriber]** \
-Messages: [ Odometry.lcm ](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/Odometry.lcm) “/odometry” \
+Messages: [Odometry.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/Odometry.lcm)
+“/odometry” \
 Publishers: jetson/filter \
 Subscribers: jetson/nav
 
 **Target List [subscriber]** \
-Messages: [ TargetList.lcm ](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/TargetList.lcm) “/target_list” \
+Messages: [TargetList.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/TargetList.lcm)
+“/target_list” \
 Publishers: jetson/percep \
 Subscribers: jetson/nav
 
-**ZED Gimbal Data [subscriber]** \
-Messages: [ ZedGimablPosition.lcm ](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/TargetList.lcm) “/zed_gimbal_data” \
-Publishers: simulators/nav, raspi/zed_gimbal, jetson/nav (TODO) \
-Subscribers: jetson/nav 
-
 **Joystick [publisher]** \
-Messages: [ Joystick.lcm ](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/Joystick.lcm) “/autonomous” \
+Messages: [Joystick.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/Joystick.lcm)
+“/autonomous” \
 Publishers: jetson/nav \
 Subscribers: jetson/teleop, simulators/nav, base_station/gui
 
 **NavStatus [publisher]** \
-Messages: [ NavStatus.lcm ](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/NavStatus.lcm) “/nav_status” \
+Messages: [NavStatus.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/NavStatus.lcm)
+“/nav_status” \
 Publishers: jetson/nav \
 Subscribers: simulators/nav, base_station/gui, jetson/science_bridge
 
+**NavStatus [publisher]** \
+Messages: [ProjectedPoints.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/ProjectedPoints.lcm)
+“/projected_points” \
+Publishers: jetson/nav
```

Comment:
base_station/gui subscribes right?

Reply:
I'm not so sure about publishers/subscribers in general tbh

Reply:
i see